CHANGES.rst

```diff
@@ -44,6 +44,9 @@ Bug Fixes
   and visibility states are now always adopted from the existing entry that would be overwritten.
   [#1538]
 
+- The top-level config will now always be returned via ``app._jdaviz_helper`` and any associated
```

Comment:
Change log is for the users and I don't think users will understand what this means for them. It should only mention the immediate bug that this fixes; i.e., an entry under bug fixes for Cubeviz that says calling cubeviz.specviz will no longer overwrites its viewer.jdaviz_helper . Do you agree?

Reply:
Sure, I can rephrase it. Let me give this another stab